src/test/java/seedu/address/storage/XmlAdaptedVisitorTest.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.storage;
+
+//@@author GAO JIAXIN666
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.visitor.Visitor;
+
+public class XmlAdaptedVisitorTest {
+    public static final String VALID_VISITOR = "GAO JIAXIN";
+    public static final String VALID_VISITOR1 = "AMY";
+
+    private Visitor visitor;
+    private Visitor visitor1;
```

Comment:
Naming convention can be improved, use a variable name like maybe anotherVisitor or differentVisitor. A name that explains what the variable is.

Reply:
Thx. Renamed